test/rb/test_rb.py

```diff
@@ -83,7 +83,56 @@ def choose_multiplier(mult_opt, len_pattern):
 
         return res
 
-    def verify_circuit(self, circ, nq, rb_opts, vec_len, result, shots):
+    @staticmethod
+    def choose_interleaved_gates(rb_pattern):
+        '''
+        :param rb_pattern: pattern for randomized benchmarking
+        :return: interleaved_gates:
+        A list of gates of random Clifford elements that
+        will be interleaved (for interleaved randomized benchmarking)
+        The length of the list would equal the length of the rb_pattern.
+        '''
+        pattern_sizes = [len(pat) for pat in rb_pattern]
+        interleaved_gates = []
+        for (_, nq) in enumerate(pattern_sizes):
```

Comment:
As we discussed let's not have a random test...pick a fixed interleaved sequence

Reply:
OK - done

Reply:
The fixed interleaved gates contain x gate on each qubit and cx gate on each pair of consecutive qubits